layout/columns2.php

```diff
@@ -38,6 +38,13 @@
 // Require own locallib.php.
 require_once($CFG->dirroot . '/theme/boost_union/locallib.php');
 
+$activitynavigation = get_config('theme_boost_union', 'activitynavigation');
```

Comment:
I have two minor comments:
* Setting the usescourseindex property here in the layout file seems slightly wrong as the property is already set in the theme's config.php file (see https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/blob/master/config.php#L49). Of course, you are overwriting the property which is set in config.php here. However, wouldn't it be able to add the logic to set the property to true or false in the theme's config.php only and omit it here?
* If you decide to keep the logic here in the layout file, please
1. move line 41 below the comment in live 43
2. add a modification comment after line 26 to ease future syncs of this file with upstream.

Reply:
Tried that (adding it directly to config.php), it did not work.

Reply:
All right. Thank you for trying it out!